spec/libraries/config_loader_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+module Alchemy
+  describe ConfigLoader do
+    context 'for elements' do
+      subject { described_class.new('elements') }
+      describe '#paths' do
+        let(:paths) { subject.paths }
+
+        it "is not empty" do
+          expect(paths).to_not be_empty
+        end
+
+        it "contains at least one option" do
+          expect(paths.length).to be > 0
+        end
+
+        it "points only to existing files" do
+          paths.each do |path|
+            expect(path.to_s).to end_with('config/alchemy/elements.yml')
+            expect(path).to be_exist
+          end
+        end
+      end
+
+      describe '#file_name' do
+        it "handle superlouas given extension" do
```

Comment:
Yeah, don't you know that animal with 2 short legs on the right side and 2 long legs on the other? ;-)
[fixing]